roles/aci/tasks/main.yml

```diff
@@ -19,10 +19,12 @@
   with_items: "{{ annotate_namespace_list }}"
   delegate_to: "{{ groups.oo_first_master.0 }}"
 - name: Get aci policy tenant name
-  shell: "grep -m1 'aci-policy-tenant' {{ aci_deployment_yaml_file }} | awk -F '\"' '{print $4}'"
+  shell: grep -m1 'aci-policy-tenant' {{ aci_deployment_yaml_file }} | awk -F '\"' '{print $4}'
   register: tenant
   delegate_to: "{{ groups.oo_first_master.0 }}"
+- name: Copy annotate_namespace.sh to /tmp
+  copy: src=annotate_namespace.sh dest=/tmp/annotate_namespace.sh mode=0777
```

Comment:
Please reformat this according to https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#Parameters-to-Ansible-modules-SHOULD-use-the-Yaml-dictionary-format-when-the-line-length-exceeds-120-characters

Reply:
Done. Thanks for the pointer.